model/src/lib.rs

```diff
@@ -192,44 +191,44 @@ impl fmt::Display for Leverage {
     }
 }
 
-// add impl's to do algebra with Usd, Leverage, and ExhangeRate as required
-impl Mul<Leverage> for Usd {
-    type Output = Usd;
+// add impl's to do algebra with Quantity, Leverage, and ExhangeRate as required
```

Comment:
```suggestion
```

this was a vague TODO without even stating that it was a TODO. I suggest getting rid of it, as of course people will add code that's needed when it's needed :-)